docs/tutorial/kymographs.rst

```diff
@@ -44,13 +72,15 @@ The kymograph can also be exported to TIFF format::
 Kymo data and details
 ---------------------
 
-We can access the raw image data as `numpy` arrays::
+We can access the raw image data as a :class:`numpy.ndarray`::
 
-    rgb = kymo.get_image("rgb")  # matrix with `shape == (h, w, 3)`
-    blue = kymo.get_image("blue")  # single color so `shape == (h, w)`
+    rgb = kymo.get_image("rgb")  # matrix with `shape == (height, width, 3 colors)`
+    blue = kymo.get_image("blue")  # single color so `shape == (height, width)`
 
     # Plot manually
-    plt.imshow(kymo.get_image("green"), aspect="auto", adjustment=lk.ColorAdjustment(0, 5))
+    plt.figure()
+    plt.imshow(kymo.get_image("green"), aspect="auto", vmax=15)
```

Comment:
Prefer `vmax` for simplicity?

Reply:
it's using `plt.imshow()` which doesn't have an `adjustment` argument 😆 just one of the many mistakes I've found now being able to actually run the tutorial code